ios/Plugin/PrivacyScreenConfig.swift

```diff
@@ -1,5 +1,7 @@
 import Foundation
 
 public struct PrivacyScreenConfig {
-    var enable = true
+    var enable = true;
+    var useImageBackground = false;
```

Comment:
This option can be removed or? If `imageName` is set, then it should be used.

Reply:
Yes it can be removed, I updated the code to just check the `imageName`